src/main/java/mate/academy/spring/dao/UserDaoImpl.java

```diff
@@ -35,6 +36,15 @@ public void add(User user) {
             }
         }
     }
+    
+    @Override
+    public Optional<User> get(Long id) {
+        try (Session session = sessionFactory.openSession()) {
+            return Optional.of(session.get(User.class, id));
+        } catch (Exception e) {
+            throw new RuntimeException("User by id: " + id + " not found", e);
```

Comment:
Message is not quite correct, if we have exception here it means problem we executing query even if user is there